frontend/src/Pages/auth/AdminSignUp.tsx

```diff
@@ -0,0 +1,91 @@
+const AdminSignin = () => {
+  return (
+    <>
+      <head>
+        <title>Student Register</title>
+      </head>
+      <form>
+        <div className="min-h-screen p-8 bg-gray-900 flex items-center justify-center">
+          <div className="container max-w-screen-lg mx-auto">
+            <div>
+              <h2 className="font-semibold text-xl text-gray-300">
+                Register as Admin
+              </h2>
+              <p className="text-gray-500 mb-6">E - LEARNING SYSTEM</p>
+
+              <div className="bg-gray-800 border-gray-400 border-[1px] rounded shadow-lg p-9 px-4 md:p-8 mb-6">
+                <div className=" grid gap-4 gap-y-2 text-sm grid-cols-1 lg:grid-cols-3">
+                  <div className="text-gray-600">
+                    <p className="font-medium text-lg text-gray-300">Admin Details</p>
+                    <p className="text-gray-400">Please fill out all the fields.</p>
+                  </div>
+
+                  <div className="md:col-span-2 text-gray-300" >
+                    <div className="grid gap-4 gap-y-3 text-sm grid-cols-2 md:grid-cols-4">
+                      <div className="md:col-span-2">
```

Comment:
For all your inputs here, please create a `components` folder and extract this into a component.

Reply:
@RDLegs 
Good morning sir, I just want to clarify , if is it for adminSignUp page only ?  or should I also implement it to student login , studentSignup and admin login page?

Reply:
If it has a similar structure in those other pages, then yes implement it for those pages as well